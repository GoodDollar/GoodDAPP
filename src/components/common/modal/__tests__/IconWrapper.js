import React from 'react'
import renderer from 'react-test-renderer'
import ImportedIconWrapper from '../IconWrapper'
import { withThemeProvider } from '../../../../__tests__/__util__'
const IconWrapper = withThemeProvider(ImportedIconWrapper)

// Note: test renderer must be required after react-native.

describe('IconWrapper', () => {
  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<IconWrapper />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
