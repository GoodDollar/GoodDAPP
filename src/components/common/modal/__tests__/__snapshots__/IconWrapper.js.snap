// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconWrapper matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1777fci r-marginVertical-1r5su4o"
  >
    <div
      className="css-view-1dbjc4n r-alignItems-1awozwy r-borderRadius-ieixw r-borderWidth-1d6rzhh r-display-6koalj r-flexDirection-18u37iz r-height-37igu r-justifyContent-1777fci r-width-19554kt"
    >
      <div
        className="css-text-901oao"
        dir="auto"
        style={
          Object {
            "color": "rgba(0,175,255,1.00)",
            "fontFamily": "gooddollar",
            "fontSize": "30px",
            "fontStyle": "normal",
            "fontWeight": "normal",
          }
        }
      >
        
      </div>
    </div>
  </div>
</div>
`;
